pages/linux/spank.md

```diff
@@ -0,0 +1,4 @@
+# spank
+
+> Spank is a component of the Slurm workload manager that allows users and administrators to create and load custom plug-ins.
+> More information: <https://slurm.schedmd.com/spank.html>.
```

Comment:
```suggestion
> SPANK - Slurm Plug-in Architecture for Node and job (K)control.
> Component of the Slurm workload manager that allows users and administrators to create and load custom plug-ins. It should not be invoked manually.
> More information: <https://slurm.schedmd.com/spank.html>.
```

While I was making this suggestion, I realized that this isn't an invokable command at all, so I would suggest dropping this page.

Reply:
Agree!